pom.xml

```diff
@@ -33,16 +37,89 @@
     </scm>
 
     <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.13.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
Why you are using both gson and jackson, you can work with json only with jackson.
Please use ObjectMapper for it.

Reply:
Let me check on this